docs/installation.md

```diff
@@ -132,6 +132,12 @@ You can run a one-off container that is immediately deleted upon exiting with th
 docker run --rm -v /etc/localtime:/etc/localtime:ro -v `pwd`/config.json:/freqtrade/config.json -it freqtrade
 ```
 
+There is known issue in OSX Docker versions after 17.09.1, whereby /etc/localtime cannot be shared causing Docker to not start. A work-around for this is to start with the following cmd. 
```

Comment:
thanks for the PR @creslinux 👍 Could you please link docker issue here for references sake and we are good to go

Reply:
The docker issue is on this link:
https://github.com/docker/for-mac/issues/2396

Thanks

Reply:
add the link to the `installation.md` please :) Sorry for not being too clear

Reply:
Ah,, Apologies. 
Should be done now - thanks.

Reply:
Ah, apologies. 
Should be done now